WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -602,6 +602,19 @@ extension OrderListViewModel {
     }
 }
 
+// MARK: - Total completed order count
+//
+extension OrderListViewModel {
+    func totalCompletedOrderCount(pageNumber: Int) -> Int? {
+        do {
+            try statusResultsController.performFetch()
```

Comment:
I don't think `performFetch` needs to be called again - when the storage is updated,  `statusResultsController`'s fetched objects are updated as well so we can just use `currentSiteStatuses` directly.

Reply:
Good point. Done in https://github.com/woocommerce/woocommerce-ios/pull/10541/commits/1948a739a2b1c1cd9439a7e17ec3bf2fd5992f14